ci/README.md

```diff
@@ -0,0 +1,54 @@
+# Ansible role wrapper around install_yamls Makefile
+
+Install_yamls makefile can be used by Dev/CI/QE as a single interface to
+deploy NextGen OpenStack.
+
+use_install_yamls role is a ansible wrapper around the install_yamls Makefile.
+It uses scripts/makefile_to_install_yamls.py script to generate the default vars
+and template file for use_install_yamls role.
+
+Run the following command to see it in action
+```
+$ rm ../roles/use_install_yamls/defaults/main.yaml ../roles/use_install_yamls/templates/install_yamls.sh.j2
```

Comment:
where are we supposed be in the path in order to run this? imho, the "../" is wrong, seeing the L15, apparently running from ci/scripts.